src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java

```diff
@@ -113,4 +120,54 @@ public AboutToStartOrSubmitCallbackResponse sendEmailNotificationOnClosingOrder(
         return AboutToStartOrSubmitCallbackResponse.builder().data(caseDataUpdated).build();
     }
 
+    @PostMapping(path = "/save-order-details", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
+    @ApiOperation(value = "Save order details")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Callback processed.", response = uk.gov.hmcts.reform.prl.models.dto.ccd.CallbackResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request")})
+    public AboutToStartOrSubmitCallbackResponse saveOrderDetails(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestBody CallbackRequest callbackRequest
+    ) throws Exception {
+        CaseData caseData = objectMapper.convertValue(
+            callbackRequest.getCaseDetails().getData(),
+            CaseData.class
+        );
+        Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
+        caseDataUpdated.put("orderCollection", manageOrderService
+            .addOrderDetailsAndReturnReverseSortedList(authorisation,caseData));
+        caseDataUpdated.remove("previewOrderDoc");
+        caseDataUpdated.remove("dateOrderMade");
+        caseDataUpdated.remove("createSelectOrderOptions");
+        return AboutToStartOrSubmitCallbackResponse.builder().data(caseDataUpdated).build();
+    }
+
+    @PostMapping(path = "/manage-orders-send-notification", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
+    @ApiOperation(value = "Callback to send FL401 application notification. ")
```

Comment:
Is this specific to FL401 ?

Reply:
This is common for both C100 and FL401

Reply:
Then please rename the API operation description and method name